addons/project/views/project_task_views.xml

```diff
@@ -651,10 +651,14 @@
                                         <field name="state" widget="project_task_state_selection" options="{'is_toggle_mode': false}"/>
                                     </div>
                                 </div>
-                                <t t-if="record.project_id.raw_value and record.subtask_count.raw_value and record.subtask_count.raw_value &gt; record.closed_subtask_count.raw_value">
+                                <!-- To do: remove me in master -->
+                                <t t-if="False and record.project_id.raw_value and record.subtask_count.raw_value and record.subtask_count.raw_value &gt; record.closed_subtask_count.raw_value">
                                     <div class="kanban_bottom_subtasks_section"/>
                                 </t>
                             </div>
+                            <t t-if="record.project_id.raw_value and record.subtask_count.raw_value and record.subtask_count.raw_value &gt; record.closed_subtask_count.raw_value">
+                                <div class="kanban_bottom_subtasks_section"/>
+                            </t>
```

Comment:
I don't agree to do that in stable, I would prefer we just move the elements in **master** and do nothing in stable because if someone did a xpath to select `div[@hasclass('kanban_bottom_subtasks_section')]` to add something next to that element, then it will no longer rendered because of our changes.

Since the fix is just design issue, I think we could say it is a limitation. I don't know if the remaining changes in that PR are depending of that changes, if yes then I would suggest to fix it in master instead.

Reply:
@xavierbol 
I have removed that commit from this PR and created a new [PR](https://github.com/odoo/odoo/pull/170841)

Thanks